tests/tasks/test_libhr_employee_appointments.py

```diff
@@ -18,39 +17,23 @@ def test_transform_libhr_employee_appointments_merge_departments(
     task_transform_libhr_employee_appointments,
 ):
     new_df = task_transform_libhr_employee_appointments.get_dataframe()
-    assert new_df.equals(
-        pd.DataFrame(
-            [
-                {
-                    "Related Employee MIT ID": 123456789,
-                    "Related Supervisor MIT ID": 444444444,
-                    "Cost Object": 555555555,
-                    "HC ID": "L-001",
-                    "Position ID": 888888888,
-                    "Related Department ID": 35.0,
-                },
-                {
-                    "Related Employee MIT ID": 987654321,
-                    "Related Supervisor MIT ID": 444444444,
-                    "Cost Object": 555555555,
-                    "HC ID": "L-100",
-                    "Position ID": 999999999,
-                    "Related Department ID": 40.0,
-                },
-                {
-                    "Related Employee MIT ID": 987654321,
-                    "Related Supervisor MIT ID": 444444444,
-                    "Cost Object": 555555555,
-                    "HC ID": "L-100",
-                    "Position ID": 999999991,
-                    "Related Department ID": np.nan,
-                },
-            ]
-        )
-    )
+    assert new_df.iloc[0]["Related Department ID"] == 35.0
+    assert new_df.iloc[1]["Related Department ID"] == 40.0
+    assert pd.isna(new_df.iloc[2]["Related Department ID"])  # no match in merge, so NULL
```

Comment:
Realized that if we are only testing the Department merge, we don't need to test the full output of the transformed data.  This removes a whack-a-mole situation where any changes need to be made here, when this test is not concerned with that.